aws-s3-transfer-manager/src/operation/download/output.rs

```diff
@@ -12,17 +12,17 @@ use crate::io::AggregatedBytes;
 
 use super::chunk_meta::ChunkMetadata;
 
-/// Stream of binary data representing an Amazon S3 Object's contents.
+/// Stream of [ChunkOutput] representing an Amazon S3 Object's contents and metadata.
 ///
 /// Wraps potentially multiple streams of binary data into a single coherent stream.
 /// The data on this stream is sequenced into the correct order.
 #[derive(Debug)]
-pub struct Body {
-    inner: UnorderedBody,
+pub struct Output {
```

Comment:
**suggestion/discuss**: Should this be consistent with the other operations output types (e.g. `DownloadOutput`)?

Reply:
Thanks, yeah, DownloadOutput makes more sense since there are different types of outputs. I have updated it.